gui/src/app/pages/HomePage/Sidebar.tsx

```diff
@@ -72,43 +71,44 @@ const Sidebar: FunctionComponent<Sidebar> = ({
         </div>
       ))}
       <Divider />
-      <List>
-        <ListItem>
-          <Button
-            variant="outlined"
-            onClick={loadProjectOpen}
-            disabled={hasUnsavedChanges}
-          >
-            Load project
-          </Button>
-        </ListItem>
 
-        <ListItem>
-          <Button
-            variant="outlined"
-            onClick={saveProjectOpen}
-            disabled={hasUnsavedChanges}
-          >
-            Save project
-          </Button>
-        </ListItem>
-        <ListItem>
-          {/* This will probably be removed or replaced in the future. It's just for convenience during development. */}
-          <Button
-            variant="outlined"
-            color="error"
-            onClick={() => {
-              const ok = window.confirm(
-                "Are you sure you want to clear all data in the editors?",
-              );
-              if (!ok) return;
-              update({ type: "clear" });
-            }}
-          >
-            Clear all
-          </Button>
-        </ListItem>
-      </List>
+      <div className="SidebarContentWrapper">
```

Comment:
Similarly, it seems like actually rendering with `ul` (which is what the MUI ListItem renders as) might be more correct? If the margin is the issue we could place the entire list in this div

Reply:
That's okay with me if you want to make that. I'm happy with how it actually renders as is, and I think in the future we'll want to revamp the sidebar to accommodate more advanced functionality.